www/scripts/codecheckerviewer/filter/DateFilter.js

```diff
@@ -214,6 +215,25 @@ function (declare, Deferred, dom, DateTextBox, TimeTextBox, SelectFilter,
           this.setDayInterval(yesterday);
           this.selectedItem = value;
           break;
+        case 'this week':
+          var today = new Date();
+          thisWeek = today.getDate() - today.getDay();
+          var monday = new Date(today.setDate(thisWeek));
```

Comment:
Not every locale has the week starting on Monday.

I'd rather see "this week", "this month" to be replaced on the gui with let's say "May 11-17", "August" and such.

Reply:
@whisperity @csordasmarton how about `last 7 days` instead of week?